### ISSUE TITLE
Feat. request: myApp.showTab(tab, force) 

### ISSUE THREAD

**Issue by petarov** [nolimits4web/Framework7#362]
**tl;dr;** Add a boolean `force` parameter to re-trigger the `show` event in case the tab is already active.

---------------
Not exactly a bug but I wanted to propose to add a force parameter to this function. Here's my case in point.

Suppose we have a specific functionality that should happen right after a tab is shown. To capsulate this one would use:

```javascript
	$$('.mytab-1').on('show', function () {
                // some sort of very specific calls
		ajaxLoad(data);
                showData(element);
	});	
```

Now, we would like this to be executed every time the user makes modifications on another page and goes back to the page where the tabs are:

```javascript
	myApp.onPageBack('config-page', function(page) {
                // Some other code ..
		myApp.showTab('.tabs.mytab-1');
	});	
```

Now this won't work because that tab is already `.active`, therefore one cannot trigger the  `show` event again.

**Comment by nolimits4web** [comment]
Ok, will add it. But you may also use it like:
```
    myApp.onPageBack('config-page', function(page) {
                // Some other code ..
        var wasActive = $('.tabs.mytab-1').hasClass('active');
        myApp.showTab('.tabs.mytab-1');
        if (wasActive) $('.tabs.mytab-1').trigger('show');
    });

**Comment by petarov** [comment]
:+1: 

Ah, dang it. My workaround was:

```javascript
$('.tabs.mytab-1').removeClass('active');
myApp.showTab('.tabs.mytab-1');
```
:)

The force param is nice to have but no biggie.
